tests/generators/ssz_generic/ssz_container.py

```diff
@@ -68,17 +68,27 @@ def valid_cases():
     for (name, (typ, offsets)) in PRESET_CONTAINERS.items():
         for mode in [RandomizationMode.mode_zero, RandomizationMode.mode_max]:
             yield f'{name}_{mode.to_name()}', valid_test_case(lambda: container_case_fn(rng, mode, typ))
-        random_modes = [RandomizationMode.mode_random, RandomizationMode.mode_zero, RandomizationMode.mode_max]
+
+        modes = [RandomizationMode.mode_random, RandomizationMode.mode_zero, RandomizationMode.mode_max]
         if len(offsets) != 0:
-            random_modes.extend([RandomizationMode.mode_nil_count,
-                                 RandomizationMode.mode_one_count,
-                                 RandomizationMode.mode_max_count])
-        for mode in random_modes:
-            for variation in range(10):
-                yield f'{name}_{mode.to_name()}_{variation}', \
-                      valid_test_case(lambda: container_case_fn(rng, mode, typ))
+            modes = list(RandomizationMode)
+
+        for mode in modes:
             for variation in range(3):
                 yield f'{name}_{mode.to_name()}_chaos_{variation}', \
+                      valid_test_case(lambda: container_case_fn(rng, mode, typ, chaos=True))
+        # Notes: Below is the second wave of iteration, and only the random mode is selected
+        # for container without offset since ``RandomizationMode.mode_zero`` and ``RandomizationMode.mode_max``
+        # are deterministic.
+        modes = [RandomizationMode.mode_random]
+        if len(offsets) != 0:
+            # Notes: ``RandomizationMode.mode_zero`` and ``RandomizationMode.mode_max`` are
+            # pseudo-random modes for containers that contains List of Bitlist
+            # (because the length of List and Bitlist are randoms).
+            modes = list(RandomizationMode)
+        for mode in modes:
+            for variation in range(10):
+                yield f'{name}_{mode.to_name()}_{variation}', \
                       valid_test_case(lambda: container_case_fn(rng, mode, typ))
```

Comment:
Simplification:
```suggestion

        # NOTE: Below is the second wave of iteration, and only the random mode is selected
        # for container without offset since ``RandomizationMode.mode_zero`` and ``RandomizationMode.mode_max``
        # are deterministic.
        # NOTE: ``RandomizationMode.mode_zero`` and ``RandomizationMode.mode_max`` are
        # pseudo-random modes for containers that contains List of Bitlist
        # (because the length of List and Bitlist are randoms).
        mode = RandomizationMode.mode_random if len(offsets) == 0 else RandomizationMode
        for variation in range(10):
            yield f'{name}_{mode.to_name()}_{variation}', \
                valid_test_case(lambda: container_case_fn(rng, mode, typ))
```

Reply:
Ah sorry, forget it, I misread! `list(RandomizationMode)`.

Reply:
I think you forget the `mode` loop, isn't it ?